src/app/static/src/app/store/metadata/metadata.ts

```diff
@@ -31,7 +31,7 @@ export const metadataGetters = {
 
         const selectedDataType = rootState.surveyAndProgram.selectedDataType;
 
-        let metadataForType = null;
+        let metadataForType: Metadata | null = null;
```

Comment:
When I ran a fresh `npm install`, `ts` compilation threw up a couple of errors. I'm actually suprised it wasn't complaining about these before, but just fixed them here.